packages/host/config/environment.js

```diff
@@ -74,7 +74,7 @@ module.exports = function (environment) {
 
   if (environment === 'production') {
     // here you can enable a production-specific feature
-    ENV.logLevels = '*=warn,current-run=error';
+    ENV.logLevels = '*=warn,current-run=error,matrix=info';
```

Comment:
This assignment means `LOG_LEVELS` has no effect in deployed environments, is that what we expect?

Reply:
I’ve since removed this as it was having no effect. I still think it would be good to reduce the Matrix logging level on startup, but for now we’ll have to do it with the logger service.